* Another Cloud9 instance created as the current one was running out of space.